### PR TITLE
Feature added to show page

### PR DESCRIPTION
**Pull request by abhilasha1405**

### CODE REVIEW

File: app/views/cats/show.html.erb [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
- <p><%= @cat.breed %></p>
- <p><%= @cat.color %></p>
- <p><%= @cat.age %></p>
-
- <div class="details">
-   <h2>A bit more about this cat</h2>
-   <p><%= @cat.description %></p>
- </div>
- <%= link_to 'Bookings', new_cat_booking_path(@cat) %>
+  <div class="booking-form">
```
Comment (patulka) [inline app/views/cats/show.html.erb:2]: why is the class called "booking-form" when there is no booking form inside?

**Review by patulka — CHANGES_REQUESTED** [review]
Please see the comment, change the class name?

Otherwise looks good :-)

**Review by patulka — CHANGES_REQUESTED** [review]
Nice! Sandra will change the class

File: app/views/cats/show.html.erb (hunk 1 above)
Comment (abhilasha1405) [reply app/views/cats/show.html.erb:2]: Sandra asked me to add . She will discuss with you about it. Also I have push another edited one ..so please ignore this one plesae.

**Pull request merged**